src/utils/constants.ts

```diff
@@ -1,6 +1,7 @@
 export const USE_MOCK_DATA = import.meta.env.VITE_USE_MOCK === 'true';
 export const ROWS_PER_PAGE_OPTIONS = [10, 25, 50, 100];
 export const DEBOUNCE_INTERVAL_INPUT = 1000;
+export const MAX_MESSAGE_LENGTH = 500;
```

Comment:
Egentlig i mot å bruke `UPPER_CASE` på sånt, da det er mer en JAVA/C# ting enn JS/TS, men siden det er sånn på de andre her blir det sikkert bare best å holde det konsekvent akkurat her. Selv om vi absolutt alle plasser ellers har `lowerCase` naming convention for `const` :P